deploy/clowdapp.yaml

```diff
@@ -158,6 +164,33 @@ objects:
     testing:
       iqePlugin: content-sources
     deployments:
+
+    - name: otel-collector
+      replicas: 1
+      webServices:
+        private:
+          enabled: true
+      podSpec:
+        image: "registry.redhat.io/rhosdt/opentelemetry-collector-rhel8"
```

Comment:
Please, let me know if we should switch to the "otel/opentelemetry-collector-contrib" image.

Reply:
I tested locally the contrib one and it works properly.